plugins/scaffolder-backend-module-sentry/src/actions/createProject.examples.test.ts

```diff
@@ -0,0 +1,232 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { registerMswTestHooks } from '@backstage/backend-test-utils';
+import { createMockActionContext } from '@backstage/plugin-scaffolder-node-test-utils';
+import { ConfigReader } from '@backstage/config';
+import { ActionContext } from '@backstage/plugin-scaffolder-node';
+import { JsonObject } from '@backstage/types';
+import { randomBytes } from 'crypto';
+import { setupServer } from 'msw/node';
+import { HttpResponse, http } from 'msw';
+import { createSentryCreateProjectAction } from './createProject';
+import yaml from 'yaml';
+import { examples } from './createProject.examples';
+
+describe('sentry:project:create action', () => {
+  const worker = setupServer();
+  registerMswTestHooks(worker);
+
+  const createScaffolderConfig = (configData: JsonObject = {}) => ({
+    config: new ConfigReader({
+      scaffolder: {
+        ...configData,
+      },
+    }),
+  });
+
+  const getActionContext = (): ActionContext<{
+    organizationSlug: string;
+    teamSlug: string;
+    name: string;
+    slug?: string;
+    authToken?: string;
+  }> =>
+    createMockActionContext({
+      workspacePath: './dev/proj',
+      logger: jest.createMockFromModule('winston'),
+      input: {
+        organizationSlug: 'org',
+        teamSlug: 'team',
+        name: 'test project',
+        authToken: randomBytes(5).toString('hex'),
+      },
+    });
+
+  it(`should ${examples[0].description}`, async () => {
+    expect.assertions(3);
+
+    let input;
+    try {
+      input = yaml.parse(examples[0].example).steps[1].input;
+    } catch (error) {
+      console.error('Failed to parse YAML:', error);
+    }
+
+    const action = createSentryCreateProjectAction(createScaffolderConfig());
+    const actionContext = getActionContext();
+
+    worker.use(
+      http.post(
+        `https://sentry.io/api/0/teams/${input.organizationSlug}/${input.teamSlug}/projects/`,
+        async ({ request }) => {
+          expect(request.headers.get('Authorization')).toBe(
+            `Bearer b15711beb516e1e910d2ede554dc1bf725654ef3c75e5a9106de9aec13d4gf93`,
+          );
+          expect(request.headers.get('Content-Type')).toBe(`application/json`);
+          await expect(request.json()).resolves.toEqual({
+            name: 'Scaffolded project B',
+          });
+          return HttpResponse.json(
+            { detail: 'project creation mocked result' },
+            { status: 201 },
+          );
+        },
+      ),
+    );
+
+    await action.handler({
+      ...actionContext,
+      input: {
+        ...actionContext.input,
+        ...input,
+      },
+    });
+  });
+
+  it(`should ${examples[0].description}`, async () => {
+    expect.assertions(3);
+
+    let input;
+    try {
```

Comment:
I appreciate that you've done this in a few tests already, but wondering if it just makes sense to simplify these statements everywhere and just remove the `try/catch` for these and throw an error. It's probably going to be a better error message than what we're going to get in these tests without it I think?

Happy to do this in a followup though - doesn't have to be done in this PR, just might make these tests a lot cleaner.

Reply:
Thanks for the suggestion, I’ll make sure to incorporate this improvement in a follow-up PR's.

In the meantime, I’d appreciate it if you could proceed with the remaining PR reviews for this issue. 

https://github.com/backstage/backstage/pulls/parmar-abhinav

Thanks again!